docs/gcp/README.md

```diff
@@ -0,0 +1,117 @@
+# NVIDIA SPARK RAPIDS on GCP Dataproc
+
+ [RAPIDS](https://rapids.ai/) suite of open source software libraries and APIs gives you the ability to execute end-to-end data science and analytics pipelines accelerated by NVIDIA GPUs. Licensed under Apache 2.0, RAPIDS is incubated by NVIDIA® based on extensive hardware and data science experience.
+ 
+ [The RAPIDS Accelerator for Apache Spark](https://github.com/NVIDIA/spark-rapids) combines the power of the RAPIDS cuDF library and the scale of the Spark distributed computing framework. The RAPIDS Accelerator library also has a built-in accelerated shuffle based on [UCX](https://github.com/openucx/ucx/) that can be configured to leverage GPU-to-GPU communication and RDMA capabilities. [NVIDIA Spark XGBoost4j](https://github.com/NVIDIA/spark-xgboost) is developed based on open sourced [XGBoost](https://github.com/dmlc/xgboost) project and leverage RAPIDS SQL plugin for data IO and Spark for scale out multi-node multi-gpu training.
+ 
+ [Google Cloud Dataproc](https://cloud.google.com/dataproc) is Google Cloud's fully managed Apache Spark and Hadoop service. In this guide, we will walk through ways to run GPU accelerated NVIDIA software on Apache Spark in Dataproc.
+ - [x] Pyspark Notebook in Dataproc JupyterLab
+ - [x] Scala Notebook in Dataproc Zeppelin
+ - [x] Submit spark jobs to Dataproc cluster
+ - [x] Dataproc Hub in AI Platform Notebook to Dataproc cluster 
+ - [ ] Spark Job to Dataproc on Google Kubernetes Engine
+ - [ ] Dataproc Workflow accelerated by NVIDIA GPU
+ 
+ ### Create Dataproc Cluster Accelerated by GPU
+ 
+ You can use [Cloud Shell](https://cloud.google.com/shell) to execute shell commands to create Dataproc cluster. Cloud Shell contains command line tools for interacting with Google Cloud Platform, including gcloud and gsutil. Alternatively, you can install [GCloud SDK](https://cloud.google.com/sdk/install) on your laptop. From the Cloud Shell, you’ll need to enable services within your project. Enable the Compute and Dataproc APIs in order to access Dataproc, and enable the Storage API as you’ll need a Google Cloud Storage bucket to house your data. This may take several minutes:
+ ```bash
+gcloud services enable compute.googleapis.com
+gcloud services enable dataproc.googleapis.com
+gcloud services enable storage-api.googleapis.com
+``` 
+
+After commandline environment is setup, account is logged in. We will now create a Dataproc cluster with following configuration (configuration matches [notebooks demo](./notebooks) we used, realistically, one can start with 2*2T4 worker nodes):
```

Comment:
nit: `After your command-line environment is setup and your account is logged in, we can now create a Dataproc with the following configuration.`